.github/workflows/update-license.yaml

```
@@ -0,0 +1,14 @@
+name: Update copyright year(s) in license file
+
+on: workflow_dispatch
```

```suggestion
on:
  workflow_dispatch:
  schedule:
    - cron: "0 3 1 1 *"
```
Ideally we should never have to remember to trigger this, so let's include the scheduled updates